systemd/openqa-enqueue-git-update-all.service

```
@@ -0,0 +1,9 @@
+[Unit]
+Description=Enqueues a git update of all tests/needles every hour.
```

This needs to be corrected as the service does not "enqueue … every hour". When it is started this still enqueues a git update but for immediate execution. Also maybe better make it clear what is the effect. And better avoid the ambiguous term "tests". How about

```suggestion
Description=Ensure a git update of all test code and needles regardless of test execution.
```

---

The "enqueuing" part is actually correct. This service does nothing more (and the gru service is actually doing the real job). So maybe:

```suggestion
Description=Enqueues a git update of all test code and needles regardless of test execution.
```

---

I thought so as well, but still changed it to @okurz' suggestion.
I think it's ok, and the script name has `enqueue` in it.